### DISCUSSION TITLE
tuiTabs selected state disappears

### DISCUSSION THREAD

**Discussion by erzenhaxhi** [taiga-family/taiga-ui#4396] [Q&A] [answered]
Hi everyone,

When the list of tabs is updated selected tab state is lost.

Playground Link: https://stackblitz.com/edit/angular-92dstq?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.module.ts

![ezgif-3-a4220cc14f](https://user-images.githubusercontent.com/40542732/236664181-8abd461d-fe7a-49d9-b769-409688edc54e.gif)

**Comment by waterplea** [comment, marked as answer]
I believe it's the same issue:
https://github.com/Tinkoff/taiga-ui/issues/4189
